Makefile

```diff
@@ -4,17 +4,20 @@ clean:
 	find . -name '*.pyc' -delete
 	find . -name __pycache__ -delete
 	find . -name '*~' -delete
+	find . -name '.coverage.*' -delete
 
 .PHONY: lint
 lint:
-	flake8 featuretools && isort --check-only featuretools
-	python docs/notebook_cleaner.py check-execution
+	isort --check-only featuretools
+	python docs/notebook_version_standardizer.py check-execution
+	black featuretools -t py39 --check
```

Comment:
Let's just do py310 and linting checking in only that ENV.

Reply:
Updated to `-t py310` which required bumping the black version as `21.9b0` did not support `-t py310`

Reply:
Thanks!